changelogs/unreleased/gh-7929-console-debugger-option.md

```diff
@@ -0,0 +1,18 @@
+## feature/debugger
```

Comment:
> > * it calls luadebug.lua::start instead of console.lua::start for interactive shell;
> What a user impact for this? Should we mention it in a doc request?

I'm confused with that comment, as I've mentioned `console.lua::start` in the commit message
itself, above doc request. Neither in `changelogs/unreleased/gh-7929-console-debugger-option.md`,
nor in `third_party/lua/README-luadebug.md` there is no reference to that internal entry point.